targets/TARGET_STM/us_ticker_32b.c

```diff
@@ -23,33 +23,30 @@
 
 TIM_HandleTypeDef TimMasterHandle;
 
-static int us_ticker_inited = 0;
-
 void us_ticker_init(void)
 {
-    if (us_ticker_inited) return;
-    us_ticker_inited = 1;
-
-    TimMasterHandle.Instance = TIM_MST;
-
-    HAL_InitTick(0); // The passed value is not used
+    /* NOTE: assuming that HAL tick has already been initialized! */
```

Comment:
This was not the case for a long time, we need to be sure that mbed_sdk_init is called before C++ objects creations happens, otherwise a timer/ticker object creation would fail.  @0xc0170  I think that the above assumption is now ensured on all toolchains thanks to the recent rework on boot sequence - do you confirm ?